src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -39,27 +41,32 @@ object Conseil extends App with LazyLogging with EnableCORSDirectives with Conse
         enableCORS {
           validateApiKey { _ =>
             logRequest("Conseil", Logging.DebugLevel) {
-              pathPrefix("tezos") {
-                tezos.route
-              } ~ pathPrefix("info") {
-                AppInfo.route
-              }
-            } ~ pathPrefix("v2") {
-              logRequest("Metadata Route", Logging.DebugLevel) {
-                pathPrefix("metadata") {
-                  platformDiscovery.route
-                }
-              } ~ logRequest("Data Route", Logging.DebugLevel) {
-                pathPrefix("data") {
-                  data.getRoute ~ data.postRoute
-                }
-              }
+              tezos.route ~
+              AppInfo.route
+            } ~
+            logRequest("Metadata Route", Logging.DebugLevel) {
+              platformDiscovery.route
+            } ~
+            logRequest("Data Route", Logging.DebugLevel) {
+              data.getRoutes ~ data.postRoute
             }
-          } ~ options {
+          } ~
+          options {
             // Support for CORS pre-flight checks.
             complete("Supported methods : GET and POST.")
           }
         }
+      } ~
```

Comment:
Can I ask for comments on matching braces when closing... I have the feeling there are mismatched braces

Reply:
I don't think they are mismatched 😉